monitoring/ceph-mixin/tests_alerts/test_alerts.yml

```diff
@@ -1482,7 +1482,7 @@ tests:
       values: '1 1 1 1 1'
    promql_expr_test:
      - expr: |
-         (predict_linear(ceph_pool_percent_used[2d], 3600 * 24 * 5) * on(pool_id)
```

Comment:
And how's it possible that this test didn't catch the error? Is it not a PromQL error but a Grafana one?

Reply:
👍🏼 thanks, @aaSharma14 !